examples/ibc/ics_test.go

```diff
@@ -0,0 +1,87 @@
+package ibc_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	interchaintest "github.com/strangelove-ventures/interchaintest/v8"
+	"github.com/strangelove-ventures/interchaintest/v8/ibc"
+	"github.com/strangelove-ventures/interchaintest/v8/testreporter"
+	"github.com/strangelove-ventures/interchaintest/v8/testutil"
+	"github.com/stretchr/testify/require"
+	"go.uber.org/zap/zaptest"
+)
+
+// This tests Cosmos Interchain Security, spinning up a provider and a single consumer chain.
+func TestICS(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping in short mode")
+	}
+
+	t.Parallel()
+
+	ctx := context.Background()
+
+	vals := 1
+	fNodes := 0
+
+	// Chain Factory
+	cf := interchaintest.NewBuiltinChainFactory(zaptest.NewLogger(t), []*interchaintest.ChainSpec{
+		{
+			Name: "ics-provider", Version: "v3.1.0",
```

Comment:
We can, however there were consumer issues with the upstream v5.x.x branch. So it would require a fork + highliner stuff since that is still a WIP upstream. [I tested this with a v5 simapp in spawn](https://github.com/rollchains/spawn/blob/5024e298a8e8ca9474f07d6758b5ad56a627e46c/simapp/interchaintest/ics_test.go). Would rather not pull that into ICT vs waiting for a fixed upstream version

Reply:
yepp i'm with you on this one, lets avoid that mess 😅